api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -689,6 +694,9 @@ public static String getOpenmrsProperty(String property) {
 		props.add(new GlobalProperty(GP_MAIL_SMTP_STARTTLS_ENABLE, "false",
 		        "Set to true to enable TLS encryption, else set to false"));
 		
+		props.add(new GlobalProperty(GP_HOST_URL, "",
+		        "The URL to redirect to after requesting for a passwordreset. Always provide a place holder in this url with name {activationKey} so that it will be substituted by the actual activation key."));
```

Comment:
The grammar is a little off around  **so that it will be substituted**,  how about **it will get substituted**?